### DISCUSSION TITLE
Can't deploy using Next.js

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#5382] [Help] [answered]
I'm using the default Next.js preset + [this website](https://github.com/X-Store-App/web) but I'm getting this error:

20:57:15.565 | > Build error occurred
-- | --
20:57:15.567 | Error: ENOENT: no such file or directory, scandir '/vercel/workpath0/blog'
20:57:15.567 | at Object.readdirSync (fs.js:955:3)
20:57:15.567 | at getStaticPaths (/vercel/workpath0/.next/serverless/pages/blog/[slug].js:857:39)
20:57:15.567 | at buildStaticPaths (/vercel/workpath0/node_modules/next/dist/build/utils.js:17:86)
20:57:15.567 | at Object.isPageStatic (/vercel/workpath0/node_modules/next/dist/build/utils.js:24:555)
20:57:15.567 | at processTicksAndRejections (internal/process/task_queues.js:97:5) {

Note that I can build this locally with 0 issues.

**Comment by ghost** [comment, marked as answer]
Fixed

**Comment by ibrahimFirat99** [comment]
How did you fix it? I am having a similar issue. It builds locally with no issues but when I upload it to Vercel I get the following error:

16:35:17.488 | errno: -2,
16:35:17.488 | code: 'ENOENT',
16:35:17.488 | syscall: 'open',
16:35:17.488 | path: '/vercel/workpath0/.next/serverless/pages/index.html'

**Reply by ghost** [reply]
This was another guy's issue when I forked his repo. I don't know.
In my case the solution was to create a file in pages/somedirectory/route.js
